src/test/java/org/folio/rest/impl/OkapiMockServer.java

```diff
@@ -91,6 +92,18 @@
 
   public static final String ERROR_TENANT = "error";
 
+  private static final String LOCATION_URI = "/locations";
+  private static final String ILL_POLICIES_URI = "/ill-policies";
+  private static final String MATERIAL_TYPES_URI = "/material-types";
```

Comment:
See my earlier comment about having to define these constants multiple times because they're private

Reply:
Thank you, corrected